help/en/html/tools/logixng/Tutorial/chapter10.shtml

```diff
@@ -0,0 +1,220 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title>LogixNG Tutorial - Chapter 10</title>
+  <meta name="author" content="Daniel Bergqvist">
+  <meta name="author" content="Dave Sand">
+  <meta name="keywords" content="jmri LogixNG tutorial modules">
+
+  <!--#include virtual="/Style.shtml" -->
+
+</head>
+
+<body>
+  <!--#include virtual="/Header.shtml" -->
+
+  <div id="mBody">
+
+    <div id="mainContent" class="no-sidebar">
+      <h1>LogixNG Tutorial - Chapter 10</h1>
+
+      <h2>Modules</h2>
+
+      <p>A module is a special ConditionalNG which is invoked from regular ConditionalNGs.
```

Comment:
A module is a special ConditionalNG which is invoked from regular ConditionalNGs **or modules**.

@dsand47  It's probably rare to use a module to call a module, but it's possible. Even recursion is possible, that the module calls itself.

Reply:
I have already used a module calling a module.